client/extensions/woocommerce/components/action-header/index.js

```diff
@@ -50,7 +50,6 @@ const ActionHeader = ( { children, breadcrumbs, isLoading } ) => {
 
 ActionHeader.propTypes = {
 	breadcrumbs: PropTypes.oneOfType( [ PropTypes.arrayOf( PropTypes.node ), PropTypes.node ] ),
```

Comment:
This `breadcrumbs` prop could also be simplified to `PropTypes.node`.

Reply:
👍  nabbed it in 10e274e8dd0893ab43c4cade4b4e1fad35fc8fd3